build/teamcity/internal/cockroach/build/ci/open-new-bazel-builder-image-pr.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+# This script expects the following env vars to be set:
+#   BRANCH [branch name to be created in fork]
+#   VERSION [new version to echo into build/.bazelbuilderversion]
+#   GH_TOKEN [github token]
+
+dir="$(dirname $(dirname $(dirname $(dirname $(dirname $(dirname "${0}"))))))"
+source "$dir/teamcity-common-support.sh"
+
+git_ssh() {
+  # $@ passes all arguments to this function to the command
+  GIT_SSH_COMMAND="ssh -i /tmp/.cockroach-teamcity-key" git "$@"
```

Comment:
I don't think this command is going to work, because the key lives in a different spot now.

I have a suggestion. How about using `GIT_SSH_COMMAND="ssh -i $dir/.cockroach-teamcity-key" git "$@"` instead. This way you don't need to move the key anywhere.